src/services/api/webforms/G1/v202401.ts

```diff
@@ -0,0 +1,461 @@
+              props: {
+                className: "w-[125px]",
+                options: [
+                  {
+                    value: "yes",
+                    label: "Yes",
+                  },
+                  {
+                    value: "no",
+                    label: "No",
+                  },
+                ],
+              },
+            },
+
+            {
+              name: "state-identifies-which-drugs-are-non-pref",
+              rhf: "Checkbox",
+              rules: { required: "* Required" },
+              dependency: {
+                conditions: [
+                  {
+                    name: "g1_cost-shar-for-drugs_state-estab-differ-cost-shar-for-prefer-and-non-prefer",
+                    type: "expectedValue",
+                    expectedValue: "yes",
+                  },
+                ],
+                effect: { type: "show" },
+              },
+              formItemClassName: "ml-[0.6rem] px-4 border-l-4 border-l-primary",
+              props: {
+                options: [
+                  {
+                    value: "true",
+                    label:
+                      "The state identifies which drugs are considered non-preferred.",
+                  },
+                ],
+              },
+            },
+
+            {
+              name: "assures-timely-process-limit-cost-shar-imposed",
+              rhf: "Checkbox",
+              rules: { required: "* Required" },
+              formItemClassName:
+                "ml-[0.6rem] px-4  border-l-4 border-l-primary",
+              dependency: {
+                conditions: [
+                  {
+                    name: "g1_cost-shar-for-drugs_state-estab-differ-cost-shar-for-prefer-and-non-prefer",
+                    type: "expectedValue",
+                    expectedValue: "yes",
+                  },
+                ],
+                effect: { type: "show" },
+              },
+              props: {
+                options: [
+                  {
+                    value: "true",
+                    label:
+                      "The state assures it has a timely process in place to limit cost sharing to the amount imposed for a preferred drug in the case of a non-preferred drug within a therapeutically equivalent or similar class of drugs if the individual's prescribing provider determines that a preferred drug for treatment of the same condition will either be less effective for the individual or will have adverse effects for the individual, or both. In such cases, reimbursement to the pharmacy is based on the appropriate cost-sharing amount.",
+                  },
+                ],
+              },
+            },
+
+            {
+              name: "all-drugs-consider-preferred-drugs",
+              rhf: "Checkbox",
+              rules: { required: "* Required" },
+              formItemClassName:
+                "ml-[0.6rem] px-4 my-2 border-l-4 border-l-primary",
```

Comment:
We use this pretty consistently in forms for "child styling" of non-child generated fields. Should we extract this somewhere? Maybe the shared-types?

Reply:
Thats a good point, lets think about this a bit